src/icalendar/cal/component_factory.py

```diff
@@ -24,9 +24,23 @@ class ComponentFactory(CaselessDict):
         >>> event_class()
         VEVENT({})
 
+    **Custom Components:**
+
```

Comment:
This is not good semantic markup because it renders as `<strong>Custom Components:</strong>`. It should be a heading or removed. However, headings in docstrings are limited to the options listed in https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections.

```suggestion
```